pytest_tests/lib/helpers/http_gate.py

```diff
@@ -23,8 +23,8 @@
 ASSETS_DIR = os.getenv("ASSETS_DIR", "TemporaryDir/")
 
 
-@allure.step("Get via HTTP Gate")
-def get_via_http_gate(
+@allure.step("Get via REST Gate")
```

Comment:
Maybe rename this file itself and the dir `http_gw` as well

Reply:
Yep. I've renamed this file and `http_gw/http_utils.py` as well